src/identity.js

```diff
@@ -464,7 +496,7 @@ export class Identity extends EventEmitter {
             let sessionData = null;
             if (this._sessionService) {
                 try {
-                    sessionData = await this._sessionService.get('/session');
+                    sessionData = await this._sessionService.get('/session', { version });
```

Comment:
`sdkVersion` would be more descrepive. Also, could it be just set as default param in https://github.com/schibsted/account-sdk-browser/blob/version-print/src/identity.js#L197 and https://github.com/schibsted/account-sdk-browser/blob/version-print/src/identity.js#L210